Package.swift

```diff
@@ -103,6 +103,14 @@ let package = Package(
         ]
     ),
 
+    // ==== Plugin for generating Swift Package from a Jar file
+    .plugin(
+        name: "JarImporterPlugin",
```

Comment:
Some food for thought -- doesn't hold up this current PR.

I wonder if we should align the words we're using. The two workflows use separate terminology. It's getting confusing remembering all the different names of tools.

In a way this is just like `jextract` but the other way (prefixing with `s` sadly gives us a bad name, so maybe we would` `swift-extract --jar` instead or something?

Alternativly dropping the jextract name would be an option, and just call it all `java2swift` and `swift2java` for the command line tools, then this would be `jar2swift --target Target <jar>` etc maybe?